src/main/rx/fport.c

```diff
@@ -278,7 +280,7 @@ static uint8_t fportFrameStatus(rxRuntimeConfig_t *rxRuntimeConfig)
                         reportFrameError(DEBUG_FPORT_ERROR_TYPE_SIZE);
                     } else {
 #if defined(USE_TELEMETRY_SMARTPORT)
-                        if (!telemetryEnabled) {
+                        if (!telemetryEnabled || (getArmingDisableFlags() & ARMING_DISABLED_GRAVEYARD)) {
```

Comment:
This will not work, since the emission of telemetry data will not be stopped, and it will most likely interfere with the second active RX.

I am currently working on code to disable emission of telemetry on the RX itself.